Marlin/temperature.cpp

```diff
@@ -487,6 +487,31 @@ int Temperature::getHeaterPower(int heater) {
     }
   }
 
+  void Temperature::checkHeatbedAutoFan() {
+    const int8_t fanPin = HEATBED_AUTO_FAN_PIN;
+    if( fanPin <= 0) return; // Early exit with no heatbed fan defined
```

Comment:
Please see https://github.com/MarlinFirmware/Marlin/wiki/DNE-Coding-Standards for code formatting guidelines.